### ISSUE TITLE
ROPE CI level: default to 89%?

### ISSUE THREAD

**Issue by DominiqueMakowski** [easystats/bayestestR#139]
Currently `equivalence_test` uses by default 0.95 CI, `rope` 0.90 and `describe_posterior` 1.00 (full ROPE).

We need to unify this.

Personally, I tend to prefer the full ROPE, for our data shows that it is more sensitive to highly significant effects and it doesn't remove anything (it is not less robust or more conservative). It is also close to a .95 or .90 equi test IF we move away from the 0% or nothing rule to a more gradual or continuous threshold (< 5%; < 2% etc.), which IMO gives a more nuanced and fine-grained understanding and interpretation.

Nevertheless, I understand that this is not, *yet*, the common choice that Bayesian experts or users would expect. So I would not mind implementing more Kruschkeyan defaults until we provide enough evidence and documentation to change. 

However, the other defaults are also a bit inconsistent. Kruschke suggests using the proportion of 95% CI. At the same time, he suggests that 95% CI is not a stable index when the number of draws is low. Hence, isn't it a bit weird to chose 95% CI in ROPE knowing that in most cases this CI is not "robust"/stable? Furthermore, since our default for CI is 90%, it seems very convoluted to provide 90% CI to describe and 95% CI in ROPE...

What do you think?

**Comment by mattansb** [comment]
Not a huge ROPE fan (BF 4eva!), but I think we should go with one of the following:
- CI, HDI and ROPE all at 90% 
- CI and HDI at 90%, ROPE at 100%

95% is for frequentists....

**Comment by DominiqueMakowski** [comment]
@strengejacke is the true ROPEist here, I am more of an effect direction kind of guy. The pd to rule them all them all; simple, clear, useful... :)

On a side note, my opinion about effect sizes (which ROPE is, strictly speaking, an index of) has changed recently. After religiously following the precept that "forget the p-value and consider the size of the effects", I've read [Funder (2019)](https://journals.sagepub.com/doi/abs/10.1177/2515245919847202) which gives a quite refreshing, interesting and counterintuitive perspective on that.

**Comment by strengejacke** [comment]
> Personally, I tend to prefer the full ROPE

A ROPE is not defined by a "level", but rather a "fixed" range based on certain rules (like 0.1 +/- * SD(y)). I guess you refer to levels of HDI or CI?

**Comment by DominiqueMakowski** [comment]
I meant the probability level of HDI yes

**Comment by strengejacke** [comment]
I personally would default to 89%, that's the threshold for which we have the most convoncing arguments.

**Comment by DominiqueMakowski** [comment]
Could you summarize (if possible) why 89 and not 90? (I can't access McElreath's statistical rethinking book from where this guideline seems to be originated 😞)

**Comment by strengejacke** [comment]
- it is lower than 95%, thus more stable
- it is as arbitrary as any other threshold
- it is the highest **prime number** that does not exceed the already unstable 95% threshold

👍

**Comment by strengejacke** [comment]
being a prime number with such attributes makes that threshold outperform any other value.

**Comment by DominiqueMakowski** [comment]
Thanks, I didn't know that. 

Should we replace all current defaults of 90% by 89% (including `hdi`)?

**Comment by strengejacke** [comment]
> Rethinking: Why 95%? The most common interval mass in the natural and social sciences is the
95% interval. This interval leaves 5% of the probability outside, corresponding to a 5% chance of the
parameter not lying within the interval (although see below). This customary interval also reflects
the customary threshold for statistical significance, which is 5% or p < 0:05. It is not easy to defend
the choice of 95% (5%), outside of pleas to convention. Ronald Fisher is sometimes blamed for this
choice, but his widely cited 1925 invocation of it was not enthusiastic:
“The [number of standard deviations] for which P = .05, or 1 in 20, is 1.96 or
nearly 2; it is convenient to take this point as a limit in judging whether a deviation
is to be considered significant or not.”
Most people don’t think of convenience as a serious criterion. Later in his career, Fisher actively
advised against always using the same threshold for significance.
So what are you supposed to do then? There is no consensus, but thinking is always a good idea.
If you are trying to say that an interval doesn’t include some value, then you might use the widest
interval that excludes the value. Often, all confidence intervals do is communicate the shape of a
distribution. In that case, a series of nested intervals may be more useful than any one interval. For
example, why not present 67%, 89%, and 97% intervals, along with the median? Why these values?
No reason. They are prime numbers, which makes them easy to remember. But all that matters is
they be spaced enough to illustrate the shape of the posterior. And these values avoid 95%, since
conventional 95% intervals encourage many readers to conduct unconscious hypothesis tests.

**Comment by strengejacke** [comment]
And another quote:

> Rethinking: What do confidence intervals mean? It is common to hear that a 95% confidence interval
means that there is a probability 0.95 that the true parameter value lies within the interval. In strict
non-Bayesian statistical inference, such a statement is never correct, because strict non-Bayesian inference forbids using probability to measure uncertainty about parameters. Instead, one should say
that if we repeated the study and analysis a very large number of times, then 95% of the computed intervals would contain the true parameter value. If the distinction is not entirely clear to you, then you
are in good company. Most scientists find the definition of a confidence interval to be bewildering,
and many of them slip unconsciously into a Bayesian interpretation.
But whether you use a Bayesian interpretation or not, a 95% interval does not contain the true
value 95% of the time. The history of science teaches us that confidence intervals exhibit chronic
overconfidence. The word true should set off alarms that something is wrong with a statement like
“contains the true value.” The 95% is a small world number (see the introduction to Chapter 2), only
true in the model’s logical world. So it will never apply exactly to the real or large world. It is what the
golem believes, but you are free to believe something else. Regardless, the width of the interval, and
the values it covers, can provide valuable advice.

(McElreath, Statistical Rethinking. page 58)

**Comment by strengejacke** [comment]
Here's a list of prime-numbers:

``` r
c(1:100)[sjstats::is_prime(1:100)]
#>  [1]  1  2  3  5  7 11 13 17 19 23 29 31 37 41 43 47 53 59 61 67 71 73 79
#> [24] 83 89 97
```

<sup>Created on 2019-05-21 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

We could use `sjstats::is_prime()` and warn users whenever they choose _no_ prime number as ci-level. :-D

**Comment by DominiqueMakowski** [comment]
Thanks a lot!

Altough... 

> They are prime numbers, which makes them easy to remember.  

![image](https://user-images.githubusercontent.com/8875533/58077922-23801c80-7be0-11e9-9abd-2f5f865dd13f.png)

**Comment by strengejacke** [comment]
> They are prime numbers, which makes them easy to remember.

Yes, I think this is one of those statistician-things, completely biased perception. ;-)

**Comment by DominiqueMakowski** [comment]
I think the main argument for 89% is indeed the "and why not 89?" and its salience-triggered emphasis on the arbitrarity of these values.

(applied to equivalence tests, it is also an argument for *full* ROPEs percentage)

Anyway, in this case, I would suggest changing all defaults of `hdi`, `ci`, `equivalence_test` and `rope` to .89. If anyone objects, let them speak now or forever hold their peace ✨

**Comment by strengejacke** [comment]
Here's the reference, I would add a sentence in the docs to say that the 89 is based on a proposal of McEalreath.

McElreath R. Statistical Rethinking. A Bayesian Course with Examples in R and Stan. Chapman and Hall, 2015

(though the 2nd edition is in progress, and you probably know https://bookdown.org/content/1850/ ?)

**Comment by DominiqueMakowski** [comment]
We must add this also to the NEWS 😀

**Comment by DominiqueMakowski** [comment]
I believe we can close this
